roles/splunk_common/tasks/check_for_required_restarts.yml

```diff
@@ -11,6 +11,9 @@
     use_proxy: no
   register: restart_required
   changed_when: restart_required.status == 200
+  until: restart_required is succeeded
+  retries: 5
```

Comment:
I'm a little worried here about the emergent behavior of this retry loop.  Like if there really is an underlying problem, it would be nice to fail faster.  Is it possible to enter this retry loop only on specific error conditions?